src/index.tsx

```
@@ -22,24 +23,34 @@ import { Web3SignerProvider } from "./context/Web3";
+export const TermsLazy = lazy(() => import("./pages/Terms"));
+
+const History = WithSuspense(HistoryLazy);
+const Rescue = WithSuspense(RescueLazy);
+const RefundEvm = WithSuspense(RefundEvmLazy);
+const RescueExternal = WithSuspense(RescueExternalLazy);
+const ClaimRescue = WithSuspense(ClaimRescueLazy);
+const RefundRescue = WithSuspense(RefundRescueLazy);
+const Privacy = WithSuspense(PrivacyLazy);
+const Terms = WithSuspense(TermsLazy);
+const NotFound = lazy(() => import("./pages/NotFound"));
+const Error = lazy(() => import("./pages/Error"));
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)